v2/common/src/main/java/com/google/cloud/teleport/v2/transforms/ValueExtractorTransform.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (C) 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.transforms;
+
+import static org.apache.beam.vendor.guava.v20_0.com.google.common.base.Preconditions.checkArgument;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.auto.value.AutoValue;
+import com.google.cloud.teleport.v2.transforms.JavascriptTextTransformer.JavascriptRuntime;
+import java.io.IOException;
+import javax.annotation.Nullable;
+import javax.script.ScriptException;
+import org.apache.beam.sdk.io.elasticsearch.ElasticsearchIO.Write.FieldValueExtractFn;
+
+/** The {@link ValueExtractorTransform}. */
```

Comment:
Add some more details in the Javadoc

Reply:
More details for the transform have been added.